src/renderer/components/+extensions/extensions.tsx

```diff
@@ -39,25 +40,20 @@ interface InstallRequestValidated extends InstallRequestPreloaded {
   tempFile: string; // temp system path to packed extension for unpacking
 }
 
-interface ExtensionState {
-  displayName: string;
-  // Possible states the extension can be
-  state: "installing" | "uninstalling";
-}
-
 @observer
 export class Extensions extends React.Component {
-  private supportedFormats = [".tar", ".tgz"];
+  private static supportedFormats = [".tar", ".tgz"];
 
-  private installPathValidator: InputValidator = {
+  private static installPathValidator: InputValidator = {
     message: <Trans>Invalid URL or absolute path</Trans>,
     validate(value: string) {
       return InputValidators.isUrl.validate(value) || InputValidators.isPath.validate(value);
     }
   };
 
-  @observable
-  extensionState = observable.map<string, ExtensionState>();
```

Comment:
I wonder if we could alternatively make this property `static` and thus singleton?

Reply:
Probably, that is basically the same thing. I just tried to follow the same style as before.